packages/loader/test/index.test.js

```diff
@@ -80,11 +80,11 @@ webpack.mdx:1:22: Unexpected end of file in expression, expected a corresponding
     }
   })
 
-  assert.not.ok(reactBuild?.hasErrors())
+  assert.ok(!reactBuild?.hasErrors())
 
   // One for ESM loading CJS, one for webpack.
   const modReact = /** @type {{default: {default: MDXContent}}} */ (
-    // @ts-expect-error file is dynamically generated
+    // @ts-ignore file is dynamically generated
```

Comment:
Does this sometimes work now?

Reply:
It always sometimes worked. First the file can’t be found. If this test is reached, the file is writen, so the file is resolved. If the assertions succeed, the file is removed, and it’s an error again.

Reply:
ah okay, perhaps we need better clean up then?

Reply:
Yes. As a follow-up, I would like to split these big test cases into smaller ones, like https://github.com/syntax-tree/estree-util-build-jsx/pull/7. That would make it simpler to add better cleanup functions.

Reply:
Yeah, sounds good